sno/resolve.py

```diff
@@ -1,53 +1,97 @@
+import copy
+import json
+from pathlib import Path
+
 import click
+import pygit2
 
 from .cli_util import MutexOption
+from .gpkg import json_ogr_to_gpkg_geom
 from .merge_util import MergeIndex, MergeContext, RichConflict
-from .exceptions import InvalidOperation, NotFound, NO_CONFLICT
+from .exceptions import InvalidOperation, NotFound, NotYetImplemented, NO_CONFLICT
 from .repo_files import RepoState
 
 
+def ungeojson_feature(feature, dataset):
+    """Given a geojson feature belonging to dataset, returns the feature as a dict containing a gpkg geometry."""
+    result = copy.deepcopy(feature['properties'])
+    if dataset.geom_column_name:
+        result[dataset.geom_column_name] = json_ogr_to_gpkg_geom(feature['geometry'])
+    return result
+
+
+def ungeojson_file(file_path, dataset):
+    """
+    Given a file containing multiple geojson features belonging to dataset,
+    returns the features as dicts containing gpkg geometries.
+    """
+    content = Path(file_path).read_text(encoding="utf-8")
+    features = json.loads(content)["features"]
```

Comment:
Use `json.load(Path(file_path))` to lower memory usage here

Reply:
although, there's only going to be two features... so... never mind? 😄

Reply:
Done